docs/crds/installEnv.yaml

```diff
@@ -14,6 +14,7 @@ spec:
   proxy:
     httpProxy: http://11.11.11.33
     httpsProxy: http://22.22.22.55
+  sshAuthorizedKey: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDphFEgnLEQknipClTiSoHpvu078bd0aTSp5WBfXvSC5nAqkprUcF/rB+iNne/MpjSONEqiUyHZayvRvVgEGfRUucAUQ89o36aJmMT8N1U85uNuPNSHVuNogyAi/pmh29C6vfs1B1qGSZRA4P/FvYn7+NdQgZQzqHxkWUDj3BqMby3+b2VKOKQ4SDHXaBSrR9BigWEWCwFJeHgTFAaEZ8p8XpPhEOhbfn9ZZ47Myq1LqfP4gw+62dBhYBNI7r44PnWbLczNLTHYUZXyappt9NBBrcwHtG+3NogDCt0eJkSYydeJh5wPQYN50kvZNJdXrMtGx9WDutTr5famFRmqcxmD'
```

Comment:
We shouldn't have secrets on GitHub

Reply:
It is just an example key in an example YAML (same as [clusterDeployment](https://github.com/openshift/assisted-service/blob/master/docs/crds/clusterDeployment-SNO.yaml#L25))
I can change it to `"your_pub_key_here"`

Reply:
This is public key, I don't think it is a problem.
Here is yours:
https://github.com/YuviGold.keys

Reply:
> It is just an example key in an example YAML (same as [clusterDeployment](https://github.com/openshift/assisted-service/blob/master/docs/crds/clusterDeployment-SNO.yaml#L25))
> I can change it to `"your_pub_key_here"`

Okay, I missed that it is documentation.
I still think that a placeholder would suites better

Reply:
Done (also for clusterDeployment)